tsconfig.json

```diff
@@ -0,0 +1,47 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "server/*": [
+        "server/*"
+      ],
+      "client/*": [
+        "client/*"
+      ]
+    },
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "jsx": "preserve",
```

Comment:
Since we're using react, I believe JSX should be set to `react` - [Read more](https://www.typescriptlang.org/docs/handbook/jsx.html#basic-usage).

Reply:
Is the preserve to pass it to babel?  because otherwise the docs would seem to suggest that @timrodz is correct here... although maybe the thing that it is passing it to is Next.js?

Reply:
Seems like it @ed42311 - next.js also forces it to be `preserve` 🙃 

```
app_1  | The following changes are being made to your tsconfig.json file:
app_1  |   - compilerOptions.jsx must be preserve
```